_includes/_clone_template.qmd

```diff
@@ -2,7 +2,7 @@
 ```{r setup, include=FALSE}
 repo_path <- system("git rev-parse --show-toplevel", intern = TRUE)
 proj_dir <- paste0(repo_path, "/guide/_viash_project_template")
-branch <- "v0.2"
+branch <- "main"
```

Comment:
Would it be possible to use a pinned version of the template? It's a serious problem if we ever have to rerun the website render and the template was changed.

Reply:
`release/0.3.x` also works, is that better? or what do you mean by pinned version?

Reply:
It looks like the only difference between the `release/0.3.0` and `main` is just a newline. So that should be fine. I suggest to use the `v0.3.0` tag though.